src/components/tl/ucp/alltoallv/alltoallv_pairwise.c

```diff
@@ -43,6 +43,25 @@ static inline size_t get_displacement(ucc_aint_t *displacements, ucc_rank_t idx,
     return ((uint32_t *)displacements)[idx];
 }
 
+static inline size_t get_total_count(ucc_count_t *counts,  ucc_tl_ucp_task_t *task)
```

Comment:
in allgatherv PR @Sergei-Lebedev adds generic ucc_coll_args_get_count/dipls routines to the common place. Should we add this get_total_count to a similar common location?

Reply:
fixed